riemann.js

```diff
@@ -1,6 +1,18 @@
 var Client = require('./riemann/client').Client;
+const e2p = require('event-to-promise');
+
+// convenience methods, how convenient.
 
-// convenience method, how convenient.
 exports.createClient = function(options, onConnect) {
   return new Client(options, onConnect);
-};
\ No newline at end of file
+};
+
+exports.createPromisifiedClient = function(options) {
```

Comment:
What about if we used options to designate the use of promises instead of a second constructor? WDYT?

Reply:
yeah, perhaps: usePromises: true ?